clients/drcachesim/drcachesim.dox.in

```diff
@@ -421,6 +422,89 @@ dcache top 10
     0x7ffcc35e7e40: 1997
 \endcode
 
+****************************************************************************
+\section sec_drcachesim_config_file Configuration File
+
+\p drcachesim supports reconfigurable cache hierarchies defined in
+a configuration file. The configuration file is a text file with the following
+formatting rules.
+
+- A comment starts with two slashes followed by one or more spaces. Anything
+after the '// ' till the end of the line is considered a comment and ignored.
+- A parameter's name and its value are listed consecutively with white space
+(spaces, tabs, or a new line) between them.
+- Parameters must be separated by white space. Including one parameter per line
+helps keep the configuration file more human-readable.
+- A cache's parameters must be enclosed inside braces and preceded by the
+cache's unique name.
+- Parameters can be listed in any order.
+- Parameters not included in the configuration file take their default values.
+- String values must not be enclosed in quotations.
+
+Supported common parameters and their values types:
+- num_cores <unsigned int>
+- line_size <unsigned int>
+- skip_refs <unsigned int>
```

Comment:
Maybe each of these should reference the corresponding runtime option, which has full documentation.  Sthg like: `Sets the "-skip_refs" option (described in \ref sec_drcachesim_ops)`.